public/components/integrations/components/create_integration_helpers.ts

```diff
@@ -282,25 +284,42 @@ export async function addIntegrationRequest(
   integration: IntegrationConfig,
   setToast: (title: string, color?: Color, text?: string | undefined) => void,
   name?: string,
-  dataSource?: string,
+  indexPattern?: string,
   workflows?: string[],
-  skipRedirect?: boolean
+  skipRedirect?: boolean,
+  dataSourceInfo?: { dataSource: string; tableName: string }
```

Comment:
For the front-end we need to add this pair of params to send to the back-end, so we do so. Todo: this method has a lot of parameters and should probably be broken up somehow.